include/flatcc/flatcc_flatbuffers.h

```diff
@@ -48,6 +48,23 @@ extern "C" {
 #define FLATBUFFERS_WRAP_NAMESPACE(ns, x) ns ## _ ## x
 #endif
 
+#include <stddef.h>
+
+/*
+ * Produce a pointer with maximum alignment to use as
+ * a placeholder when FLATCC_TOLERATE_MISALIGNED_EMPTY_VECTORS
+ * is specified.
+ */
+static inline const void* max_aligned_ptr(void)
```

Comment:
Since it seems that avoiding *materialization* of misaligned pointers (and in general modifying the reader) is not essential for this PR, I'll just revert all those changes.